cpp/src/parquet/encoding.h

```diff
@@ -466,4 +467,9 @@ std::unique_ptr<typename EncodingTraits<DType>::Decoder> MakeTypedDecoder(
   return std::unique_ptr<OutType>(dynamic_cast<OutType*>(base.release()));
 }
 
+PARQUET_EXPORT
```

Comment:
It's ok but just curious why it's export?

Reply:
https://github.com/apache/arrow/pull/40957#discussion_r1548864840 suggested fix for windows CI issue. Seems to have worked.

Reply:
If the purpose of adding it to header file is for testing, I'd suggest adding a `encoding_internal.h` instead. Then we don't need to include parquet/properties.h as well.